Fix/update docs alignment matrix control (#34624)
WordPress/gutenberg: packages/components/src/alignment-matrix-control/README.md

## Usage

```jsx

> import { __experimentalAlignmentMatrixControl as AlignmentMatrixControl } from '@wordpress/components';
> import { useState } from '@wordpress/element';
> 
> const Example = () => {
> 	const [alignment, setAlignment] = useState('center center');
> 
> 	return (
> 		<AlignmentMatrixControl
> 			value={alignment}
> 			onChange={(newAlignment) => setAlignment(newAlignment)}
> 		/>
> 	);
> };

Reviewer: Very very minor, but — it'd be great if the example snippet followed the repo's [prettier formatting rules](https://github.com/WordPress/gutenberg/blob/375eaa0d870282e12cd461a53463d3e10e776c41/packages/prettier-config/lib/index.js)

```suggestion
import { __experimentalAlignmentMatrixControl as AlignmentMatrixControl } from '@wordpress/components';
import { useState } from '@wordpress/element';

const Example = () => {
	const [ alignment, setAlignment ] = useState( 'center center' );
	return (
		<AlignmentMatrixControl
			value={ alignment }
			onChange={ ( newAlignment ) => setAlignment( newAlignment ) }
		/>
	);
};
```

```

## Props

The component accepts the following props:
### className

The class that will be added with "component-alignment-matrix-control" to the classes of the wrapper <Composite/> component.

> If no className is passed only "component-alignment-matrix-control" is used.

Reviewer: I know that some older READMEs have a similar paragraph, but from now on let's not mention the `"component-alignment-matrix-control"` className in public docs

-   Type: `String`
-   Required: No

### id

Unique ID for the component.
-  Type: `String`
-  Required: No

> ### label

Reviewer: Tiny nitpick: most headers used for props are missing an empty line above them

```suggestion

### label
```

> If provided, sets the aria-label attribute of the wrapper <Composite/> component.

Reviewer: Nit: 

```suggestion
Accessible label. If provided, sets the `aria-label` attribute of the underlying <Composite/> component.
```

-   Type: `String`
-   Required: No
-   Default: `Alignment Matrix Control`

> ### defaultValue

Reviewer: Can we also add the `value` prop to the docs? It looks like it's currently missing

If provided, sets the default alignment value.
- Type: `String`
- Required: No
- Default: `center center`

### onChange

A function that receives the updated alignment value.

> -   Type: `function`

Reviewer: Could we replace types to be closer to how TypeScript types are described? 

In this case, it could be something like `( nextValue: string ) => void`

Other changes would be:

- from `String` to `string`
- from `Number` to `number`

-   Required: No

> -   Default: `noop`

Reviewer: I know that the default value for this prop is a `noop` function, but I'd remove this line from the README, as it may confuse a reader instead of adding value
